editor/plugins/expression_evaluator_editor_plugin.cpp

```diff
@@ -60,14 +64,15 @@ void EditorExpressionEvaluator::_evaluate() {
 		return;
 	}
 
-	if (!editor_debugger->is_session_active()) {
+	if (!session->is_active()) {
 		return;
 	}
 
 	Array expr_data;
 	expr_data.push_back(expression);
-	expr_data.push_back(editor_debugger->get_stack_script_frame());
-	editor_debugger->send_message("evaluate", expr_data);
+	// expr_data.push_back(editor_debugger->get_stack_script_frame());
```

Comment:
I couldn't find a way to access ScriptEditorDebugger and this method doesn't look exposed anywhere.